public/utils/validation.ts

```diff
@@ -15,7 +15,7 @@ export const NAME_REGEX = new RegExp(/^[a-zA-Z0-9 _-]{5,50}$/);
 export const DETECTION_NAME_REGEX = new RegExp(/^[a-zA-Z0-9_.-]{5,50}$/);
 
 export const CONDITION_REGEX = new RegExp(
```

Comment:
Let's rename this to describe what condition it is used for

Reply:
Done.